src/SWOTRiver/products/rivertile.py

```diff
@@ -634,17 +634,26 @@ def update_from_pixc(self, pixc_file, index_file):
         """Adds more datasets from pixc_file file using index_file"""
         pixc_vec = L2PIXCVector.from_ncfile(index_file)
 
+        #~ pixc2rivertile_map = {
+            #~ '/pixel_cloud/solid_earth_tide': 'earth_tide',
+            #~ '/pixel_cloud/pole_tide': 'pole_tide',
+            #~ '/pixel_cloud/loading_tide_sol1': 'load_tide',
+            #~ '/pixel_cloud/model_dry_tropo_cor': 'dry_trop_c',
+            #~ '/pixel_cloud/model_wet_tropo_cor': 'wet_trop_c',
+            #~ '/pixel_cloud/model_iono_ka_cor': 'iono_c',
+            #~ '/pixel_cloud/xover_height_cor': 'xovr_cal_c',
+            #~ '/tvp/time': 'time',
+            #~ '/tvp/time_tai': 'time_tai'}
+            
         pixc2rivertile_map = {
             '/pixel_cloud/solid_earth_tide': 'earth_tide',
             '/pixel_cloud/pole_tide': 'pole_tide',
             '/pixel_cloud/loading_tide_sol1': 'load_tide',
             '/pixel_cloud/model_dry_tropo_cor': 'dry_trop_c',
             '/pixel_cloud/model_wet_tropo_cor': 'wet_trop_c',
             '/pixel_cloud/model_iono_ka_cor': 'iono_c',
-            '/pixel_cloud/xover_height_cor': 'xovr_cal_c',
-            '/tvp/time': 'time',
-            '/tvp/time_tai': 'time_tai'}
-
+            '/pixel_cloud/xover_height_cor': 'xovr_cal_c'}
+            
```

Comment:
no I need this. If it doesn't work give me an example and find out why I am not removing this